app/src/main/java/com/sopt/now/compose/HomeScreen.kt

```diff
@@ -1,38 +1,52 @@
 package com.sopt.now.compose
 
+import android.util.Log
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.mutableStateOf
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import com.sopt.now.compose.data.DTO.response.ResponseUserInfoDataDto
+import com.sopt.now.compose.data.DTO.response.ResponseUserInfoDto
+import com.sopt.now.compose.data.ServicePool
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
 
 @Composable
-fun HomeScreen(nickname: String?, mbti: String?) {
+fun HomeScreen(userId: String) {
     val homeViewModel = HomeViewModel()
+
     val friendsList = homeViewModel.getFriendsData()
+    val user = homeViewModel.user.collectAsState()
+
+    homeViewModel.getUserData(userId)
 
     LazyColumn {
         item {
             UserItem(
                 painter = R.drawable.ic_launcher_foreground,
                 contentDescription = "User Profile Image",
                 imageSize = 70.dp,
-                nickname = nickname ?: "닉네임",
-                mbti = mbti ?: "mbti",
+                nickname = user.value?.nickname ?: "닉네임",
+                phone = user.value?.phone ?: "phone",
```

Comment:
null일 때 처리를 잘 해주셨네요!

전반적으로 안드로이드 앱에서 사용하는 문구는 strings.xml이라는 파일에서 관리해서 그 쪽 파일들에 변수를 선언해두고 `stringResource(id = R.string.nickname)` 이런식으로 가져오면 좋을 것 같네요. 반영하면 아래처럼 될거에요!
```suggestion
                nickname = user.value?.nickname ?: stringResource(id = R.string.nickname),
                phone = user.value?.phone ?: stringResource(id = R.string.phone),
```